src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/ReadFromTheQueueTest.java

```diff
@@ -1,42 +1,13 @@
 package uk.gov.hmcts.reform.bulkscan.orchestrator;
 
-import com.microsoft.azure.servicebus.ClientFactory;
-import com.microsoft.azure.servicebus.IMessageReceiver;
 import com.microsoft.azure.servicebus.Message;
-import com.microsoft.azure.servicebus.QueueClient;
-import com.microsoft.azure.servicebus.ReceiveMode;
-import com.microsoft.azure.servicebus.primitives.ConnectionStringBuilder;
-import com.typesafe.config.Config;
-import com.typesafe.config.ConfigFactory;
-import org.junit.Before;
 import org.junit.Test;
 
 import java.util.UUID;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class ReadFromTheQueueTest {
-
-    private IMessageReceiver readClient;
-    private QueueClient writeClient;
-    private int readInterval;
-
-    @Before
-    public void setUp() throws Exception {
-        Config conf = ConfigFactory.load();
-
-        this.readInterval = conf.getInt("queue.read-interval");
-
-        this.readClient = ClientFactory.createMessageReceiverFromConnectionString(
-            conf.getString("queue.conn-strings.read"),
-            ReceiveMode.PEEKLOCK
-        );
-
-        this.writeClient = new QueueClient(
-            new ConnectionStringBuilder(conf.getString("queue.conn-strings.write")),
-            ReceiveMode.PEEKLOCK
-        );
-    }
+public class ReadFromTheQueueTest extends BaseTest {
```

Comment:
Yup Ive always found BaseTest classes end up being a monolith dumping ground.
try break out functional components to inject.